src/Yesod/Auth/Simple.hs

```diff
@@ -564,16 +565,30 @@ setError = setSession "error"
 clearError :: AuthHandler a ()
 clearError = deleteSession "error"
 
+getEmail :: AuthHandler a (Maybe Text)
+getEmail = do
+  mEmail <- lookupSession "email"
```

Comment:
Can this session key clash with client code?

Reply:
Maybe I should call it `yesod-auth-simple-email` if that is what you are suggesting?

Reply:
Yes, and yes! For globals we may want to document them as well.